src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -2,12 +2,12 @@
 import * as React from 'react';
 import {Text, View, StyleSheet, TouchableOpacity, Pressable, Platform} from 'react-native';
 import ToastMessage from 'react-native-toast-message';
-import TextAreaChat from '../../../components/TextAreaChat';
-import ToggleSwitch from '../../../components/ToggleSwitch';
+import CheckBox from '@react-native-community/checkbox';
 import {profileSettingsDMpermission} from '../../../service/profile';
 import {addDotAndRemoveNewline} from '../../../utils/string/TrimString';
 import {COLORS} from '../../../utils/theme';
-import {useDynamicColors} from '../../../hooks/useToggleColors';
+import {PencilIcon} from '../../../assets';
+import {Divider} from '../../../components/Divider';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code has replaced `TextAreaChat` and `ToggleSwitch` components with `CheckBox` from `@react-native-community/checkbox`. This change seems to be a part of the UI redesign. However, it's important to ensure that the `CheckBox` component provides all the necessary functionality and user experience that the previous components did.



<!-- This is an auto-generated comment by OSS CodeRabbit -->